package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "azure-iot-toolkit",
   "displayName": "Azure IoT Hub",
-  "description": "This extension is now a part of Azure IoT Tools extension pack. We highly recommend installing Azure IoT Tools to get full capabilities for Azure IoT development.",
+  "description": "Interact with Azure IoT Hub, IoT Device Management, IoT Edge Management, IoT Hub Device Simulation, IoT Hub Code Generation and IoT Hub Device Provisioning Service (DPS). This extension is now a part of Azure IoT Tools extension pack. We highly recommend installing Azure IoT Tools to get full capabilities for Azure IoT development.",
```

Comment:
The new description should be moved to the beginning...

Reply:
updated